tests/tests_twobody/test_perturbations.py

```diff
@@ -191,7 +191,7 @@ def test_atmospheric_drag_exponential():
 
     # parameters of a body
     C_D = 2.2  # dimentionless (any value would do)
-    A_over_m = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value / 100  # km^2/kg
+    A_over_m = ((np.pi / 4.0) * (u.m ** 2)).to(u.km ** 2).value / 100 * u.kg  # km^2/kg
```

Comment:
Isn't it better to do something like

```python
A_over_m = ((np.pi / 4 * u.m ** 2) / (100 * u.kg)).to_value(u.km ** 2 / u.kg)
```

? (This applies everywhere)

Reply:
Yes i was also doing this only. Let me see if it works or not